analytics_data_api/v0/views/learners.py

```diff
@@ -36,24 +36,29 @@ class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
 
     **Response Values**
 
-        Returns the learner metadata and engagement data:
+        Returns metadata and engagement data for the learner.
 
             * username: User's username.
-            * enrollment_mode: Enrollment mode (e.g. "honor).
+            * enrollment_mode: Enrollment mode (for example, "verified").
             * name: User's full name.
-            * email: User's email.
-            * segments: Classification for this course based on engagement, (e.g. "has_potential").
+            * email: User's email address.
+            * segments: Classification, based on engagement, of this learner's
+              work in this course (for example, "highly_engaged" or
+              "struggling").
             * engagements: Summary of engagement events for a time span.
-                * videos_viewed: Number of times a video was played.
-                * problems_completed: Unique number of problems completed.
-                * problems_attempted: Unique number of problems attempted.
-                * discussions_contributed: Number of discussions (e.g. forum posts).
+                * videos_viewed: Number of times any course video was played.
+                * problems_completed: Number of unique problems the learner
+                  answered correctly.
+                * problems_attempted: Number of unique problems attempted.
+                  This is a count of the different problems the learner tried.
+                * discussions_contributed: Number of posts, responses, or
+                  comments the learner contributed to course discussions.
 
     **Parameters**
 
         You can specify the course ID for which you want data.
 
-        course_id -- The course within which user data is requested.
+        course_id -- The course for which user data is requested.
```

Comment:
This is the identifier, not the course name, right? Would an example be helpful? For example, RELI157x.